modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItSqlUsesKeyValuePutTest.java

```diff
@@ -65,6 +66,69 @@ void insertConstantSimpleKey() {
         }
     }
 
+    @Test
+    void insertConstantReversFieldOrder() {
+        for (int i = 0; i < TABLE_SIZE; i++) {
+            assertQuery(format("INSERT INTO simple_key (val, id) VALUES ({}, {})", i * 10, i))
```

Comment:
probably we need to append smth more complex ? for example:
`assertQuery(format("INSERT INTO simple_key VALUES ({}::smallint + SUBSTRING('1234567', 1, 1), {})", i, i))`

Reply:
new test has been added (`ItSqlUsesKeyValuePutTest#insertExpressionSimpleKey`)